recipe/meta.yaml

```diff
@@ -15,7 +15,7 @@ source:
 build:
   number: 0
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
-  skip: true  # [win]
+  skip: true  # [win and py<=35]]
```

Comment:
The logic here is filtering: the skip statement will only activate for builds that are on Windows *and* use Python <= 3.5. You want `or` here.

Reply:
oh, of course. Thank you!